style.css

```diff
@@ -522,8 +617,94 @@
     height: 48px;
   }
 
-  .mobile-squeleton .header .header-menu {
-    display: none;
+  .show {
+    display: block;
+  }
+
+  /* Mobile menu bar  */
+
+  .hamburgeur {
+    display: block;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: -15px;
+    bottom: 0;
+    height: 100vh;
+    width: 100vw;
+    background-color: #6070ff;
+    mix-blend-mode: darken;
+    background-image: url('images/menu-barBg.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    flex-direction: column;
+    z-index: 10;
+    opacity: 1;
+    -webkit-animation: start 0.8s ease-in;
+    animation: start 0.8s ease-in;
+  }
```

Comment:
- [**OPTIONAL**] I think it would be better if you remove the **scrolling**  when you are watching the **Mobile menu**, this will prevent users to interact with content in the sub-background.  You can use a CSS property `overflow: hidden; doing that, will make your mobile menu correspond to the original design and behaviors professional.